engine/apps/metrics_exporter/helpers.py

```diff
@@ -53,17 +53,19 @@ def is_allowed_to_start_metrics_calculation(organization_id, force=False) -> boo
     """Check if metrics_cache_timer doesn't exist or if recalculation was started by force."""
     recalculate_timeout = get_metrics_recalculation_timeout()
     metrics_cache_timer_key = get_metrics_cache_timer_key(organization_id)
-    metrics_cache_timer = cache.get(metrics_cache_timer_key)
-    if metrics_cache_timer:
-        if not force or metrics_cache_timer.get("forced_started", False):
-            return False
-        else:
-            metrics_cache_timer["forced_started"] = True
-    else:
-        metrics_cache_timer: RecalculateMetricsTimer = {
+
+    metrics_cache_timer: RecalculateMetricsTimer = cache.get(
+        metrics_cache_timer_key,
+        {
             "recalculate_timeout": recalculate_timeout,
             "forced_started": force,
-        }
```

Comment:
refactored this to simply provide a default value of: 
```python
{
    "recalculate_timeout": recalculate_timeout,
    "forced_started": force,
}
```

to the `cache.get` call, rather than needing an `else` clause (otherwise it complains about `no-redef`)